EIPS/eip-7733.md

```diff
@@ -0,0 +1,60 @@
+---
+eip: 7733
+title: Deactivate EIP-158
+description: Deactivates EIP-158 in order to avoid conflicts in a context in which statelessness and EIP-7702 are active
```

Comment:
I'd recommend removing EIP-7702 from the description. Reading 7702 is not a requirement to understand this proposal; it's more a motivating use case.